### ISSUE TITLE
Laravel 9 Compatability

### ISSUE THREAD

**Issue by poxin13** [cknow/laravel-money#106]
Not able to update when using requiring laravel/framework ^9.0

```
- Root composer.json requires cknow/laravel-money ^6.2 -> satisfiable by cknow/laravel-money[v6.2.0].
    - Conclusion: don't install laravel/framework v9.0.0-beta.2 (conflict analysis result)
    - Conclusion: don't install laravel/framework v9.0.0-beta.3 (conflict analysis result)
    - Conclusion: don't install laravel/framework v9.0.0-beta.4 (conflict analysis result)
    - Conclusion: don't install laravel/framework v9.0.0-beta.5 (conflict analysis result)
    - Conclusion: don't install laravel/framework v9.0.0 (conflict analysis result)
    - Conclusion: don't install laravel/framework v9.0.1 (conflict analysis result)
    - Conclusion: don't install laravel/framework v9.0.2 (conflict analysis result)
    - Conclusion: don't install laravel/framework v9.0.0-beta.1 (conflict analysis result)
    - cknow/laravel-money v6.2.0 requires illuminate/view ^7.0|^8.0 -> satisfiable by illuminate/view[v7.0.0, ..., 7.x-dev, v8.0.0, ..., 8.x-dev].
    - Only one of these can be installed: illuminate/view[v7.0.0, ..., 7.x-dev, v8.0.0, ..., 8.x-dev], laravel/framework[v9.0.0-beta.1, ..., 9.x-dev]. laravel/framework replaces illuminate/view and thus cannot coexist with it.
    - Root composer.json requires laravel/framework ^9.0 -> satisfiable by laravel/framework[v9.0.0-beta.1, ..., 9.x-dev].
```

**Comment by edwinvdpol** [comment]
@ricardogobbosouza Would you be so kind to make a new release?

This will support Laravel 9, and this is the only package that is preventing us from upgrading.

Thank you for your time, and for this package!

-- Edwin

**Comment by ricardogobbosouza** [comment]
Hi @edwinvdpol, @poxin13 

Released [v6.3.0](https://github.com/cknow/laravel-money/releases/tag/v6.3.0)

**Comment by edwinvdpol** [comment]
Thank you, appreciated!
